razer_cli/listing/lister.py

```diff
@@ -0,0 +1,133 @@
+from argparse import Namespace
+
+from razer_cli import settings, util
+
+
+def list_devices(device_manager, args: Namespace):
```

Comment:
Someone with a Python version older than 3.9 please check if this `Namespace` type annotation works. I've used it multiple times throughout this PR. If there is any doubt I'll remove it.

@GM-Script-Writer-62850 what do you use?